src/components/ReviewCard.jsx

```diff
@@ -17,23 +17,33 @@ const ReviewCard = ({
   const [votes, setVotes] = useState(vote);
   const [isError, setIsError] = useState(false);
   const [hasClicked, setHasClicked] = useState(false);
-
+  const [hasClickedDownVote, setHasClickedDownVote] = useState(false);
   useEffect(() => {
     fetchVotes(id).then((currVotes) => {
       setVotes(currVotes);
     });
   }, []);
 
-  const votesHandler = () => {
+  const upVotesHandler = () => {
     setVotes((currentVotes) => currentVotes + 1);
     setIsError(false);
     setHasClicked(true);
-    increaseVotes(id).catch(() => {
+    increaseVotes(id, 1).catch(() => {
       setVotes((currentVotes) => currentVotes - 1);
       setIsError(true);
     });
   };
 
+  const downVotesHandler = () => {
+    setVotes((currentVotes) => currentVotes - 1);
+    setIsError(false);
+    setHasClickedDownVote(true);
+    increaseVotes(id, -1).catch(() => {
+      setVotes((currentVotes) => currentVotes + 1);
+      setIsError(true);
+    });
+  };
+
```

Comment:
These two functions could be one function

Reply:
I like how you have handled these with being able to vote each only once. if you think about most websites that have both up and down votes it will tend to remove the up vote if you change to a down vote and vice versa. this could be something to consider down the line